diff --git a/common/FrEvent.i b/common/FrEvent.i
index b97e5d5..894b375 100644
--- a/common/FrEvent.i
+++ b/common/FrEvent.i
@@ -30,7 +30,7 @@
 #define FR_EVENT_I
 
 %module(package="LDAStools") frameCPP
-
+%include <attribute.i>
 %include "std_pair.i"
 %include "std_string.i"
 %include "std_vector.i"
diff --git a/common/FrHistory.i b/common/FrHistory.i
index 139f007..7867ab8 100644
--- a/common/FrHistory.i
+++ b/common/FrHistory.i
@@ -30,7 +30,7 @@
 #define FR_HISTORY_I
 
 %module(package="LDAStools") frameCPP
-
+%include <attribute.i>
 %{
 #include "ldastoolsal/types.hh"
 #include "framecpp/FrHistory.hh"
diff --git a/common/FrProcData.i b/common/FrProcData.i
index 3774754..7b76dbd 100644
--- a/common/FrProcData.i
+++ b/common/FrProcData.i
@@ -30,7 +30,7 @@
 #define FR_PROC_DATA_I
 
 %module(package="LDAStools") frameCPP
-
+%include <attribute.i>
 %{
 #include "ldastoolsal/types.hh"
 #include "framecpp/FrProcData.hh"
diff --git a/common/FrTable.i b/common/FrTable.i
index 608aaa7..315cc8d 100644
--- a/common/FrTable.i
+++ b/common/FrTable.i
@@ -30,7 +30,7 @@
 #define SWIG__COMMON__FR_TABLE_I
 
 %module(package="LDAStools") frameCPP
-
+%include <attribute.i>
 %{
 #include "ldastoolsal/types.hh"
 #include "framecpp/FrTable.hh"
diff --git a/common/FrVect.i b/common/FrVect.i
index c68fcbb..34e7f3a 100644
--- a/common/FrVect.i
+++ b/common/FrVect.i
@@ -36,7 +36,7 @@
  */
 
 %module(package="LDAStools") frameCPP
-
+%include <attribute.i>
 %feature("autodoc","1");
 
 %include <boost_shared_ptr.i>
diff --git a/common/GPSTime.i b/common/GPSTime.i
index 642ab14..daf5154 100644
--- a/common/GPSTime.i
+++ b/common/GPSTime.i
@@ -30,7 +30,7 @@
 #define GPSTIME_I
 
 %module(package="LDAStools") frameCPP
-
+%include <attribute.i>
 %{
 #include "framecpp/GPSTime.hh"
 
